src/Components/Accordion.tsx

```diff
@@ -0,0 +1,35 @@
+import type { ComponentChildren } from 'preact';
+import { useState } from 'preact/hooks';
+
+type AccordionProps = {
+    id: string;
+    title: string;
+    expanded: boolean;
```

Comment:
We should rename that to `expandedInitially` or something like that.

Reply:
Yes, please do that.